docs/test_env.conf

```diff
@@ -31,7 +31,7 @@ smtp_config:
   from_name: your value here
   username: your value here
   password: your value here
-tst_aoc:
+tst_oncloud:
```

Comment:
Why revert this back to `tst_oncloud`? Isn't it more consistent to use `aoc` everywhere?

Reply:
This is only a name reference, whatever reference name is ok.
the default section has:
aoc: tst_oncloud
and then later the reference is defined:
tst_oncloud: ...